.github/workflows/terraform-plan-apply.yml

```diff
@@ -163,4 +165,34 @@ jobs:
                           -var="trial_sc_gateway_image_tag=${{ matrix.trial_sc_gateway_image_tag }}" \
                           -var="trial_sc_discovery_image_tag=${{ matrix.trial_sc_discovery_image_tag }}" \
                           -var="trial_sc_config_image_tag=${{ matrix.trial_sc_config_image_tag }}" \
-                          -auto-approve
\ No newline at end of file
+                          -auto-approve
+
+          # '-raw' is important so no other characters will be added
+          ui_storage_conn_string=$(terraform output -raw ui_conn_string)
+          # create an env var from the connection string, so will be available for the later steps
+          echo "ui_storage_conn_string=$ui_storage_conn_string" >> $GITHUB_ENV
+          echo "trial_name=$matrix.trial_name" >> $GITHUB_ENV
+
+          ## DO NOT DELETE ##
+          # mask this value so this sensitive data won't be logged.
+          echo "::add-mask::$ui_storage_conn_string"
+
+      # Get release zip file
+      - name: UI component - Get assets zip
+        run: |
+          wget https://github.com/NDPH-ARTS/mts-trial-ui/releases/download/latest/all_assets.zip
+
+      # Unzip & Customzie (replace values)
+      - name: UI component - Unzip and modify
+        run: |
+          unzip all_assets.zip -d ui_component
+
+          issuer="https://login.microsoftonline.com/5d23383f-2acb-448e-8353-4b4573b82276/v2.0"
+          sed -i 's/{{clientId}}"/$ARM_CLIENT_ID/' ui_component/index.html
```

Comment:
This should be different than the one terraform uses... probably get this also from the config json.